tools/gen_android_buildconfig.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import argparse
+import os
+import sys
+
+BUILD_CONFIG_TEMPLATE = """
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+
+// THIS FILE IS AUTO_GENERATED
+// DO NOT EDIT THE VALUES HERE - SEE $flutter_root/tools/gen_android_buildconfig.py
+package io.flutter;
+
+public final class BuildConfig {{
+  public final static boolean DEBUG = {0};
+  public final static boolean PROFILE = {1};
+  public final static boolean RELEASE = {2};
```

Comment:
nit: Would also be a bit safer to add a private constructor here, just to make sure this is only used for the constants.

Reply:
Done